tests/test_dse.py

```diff
@@ -13,53 +13,41 @@
 from devito.stencil import Stencil
 from devito.visitors import FindNodes
 from examples.seismic.acoustic import AcousticWaveSolver
-from examples.seismic import Model, PointSource, Receiver
-from examples.seismic.tti.tti_example3D import setup
+from examples.seismic import demo_model, RickerSource, GaborSource, Receiver
+from examples.seismic.tti import AnisotropicWaveSolver
 
 
 # Acoustic
 
 def run_acoustic_forward(dse=None):
     dimensions = (50, 50, 50)
-    origin = (0., 0., 0.)
```

Comment:
In outward-facing examples, yes, maybe, but probably not in internal tests like this.